server/config/config.go

```diff
@@ -348,7 +358,14 @@ func (c TLSConfig) Validate() error {
 
 // DefaultMemIAVLConfig returns the default MemIAVL configuration
 func DefaultMemIAVLConfig() *MemIAVLConfig {
-	return &MemIAVLConfig{memiavlcfg.DefaultMemIAVLConfig()}
+	return &MemIAVLConfig{memiavlcfg.MemIAVLConfig{
+		Enable:             false,
+		ZeroCopy:           false,
```

Comment:
Set this two as constants as well please 🙏 - At the beginning of the Config file we set up all the Default constants that way we remain consistent